packages/editor/src/extensions/code-block/highlighter.ts

```diff
@@ -124,24 +124,22 @@ export function HighlighterPlugin({
           if (!pluginState) return;
 
           const changedBlocks: Set<string> = new Set();
-          for (const blockKey in pluginState.languages) {
-            if (HIGHLIGHTED_BLOCKS.has(blockKey)) continue;
 
+          for (const blockKey in pluginState.languages) {
```

Comment:
Why this change?

Reply:
I did not change this line. Its the same.

Reply:
Why is it showing in the diff?

Reply:
It is some kind of glitch probab it is not showing in the diff in my view.

Reply:
OK.